[10.0] [BACKPORT] l10n_ch_qr_bill_scan (#561)
OCA/l10n-switzerland: l10n_ch_qr_bill_scan/readme/USAGE.rst

> To import your invoices Launch the wizard `Invoicing` -> `Vendors` -> `Import Vendor Bill`.

Reviewer: ```suggestion
To import your invoices, launch the wizard `Invoicing` -> `Vendors` -> `Import Vendor Bill`.
```

> In there you can either directly the QR code of the QR-bill or you can import a PDF/image file.

Reviewer: Missing word, here. "import" ? "scan" ? 
```suggestion
In there you can either directly import the QR code of the QR-bill or you can import a PDF/image file.
```

> If it's the first time you import a Bill from a Vendor the wizard will ask you to create the partner and to set a import configuration for that partner

Reviewer: ```suggestion
If it's the first time you import a Bill from a Vendor the wizard will ask you to create the partner and to set an import configuration for that partner
```

> A configuration let you set which expense account you want the invoice to be imported with.

Reviewer: ```suggestion
A configuration lets you set which expense account you want the invoice to be imported with.
```

> To quickly import a serie of invoice, use the button `Import Next Invoice` from the Vendor bill form.

Reviewer: ```suggestion
To quickly import a series of invoices, use the button `Import Next Invoice` from the Vendor bill form.
```
